README.rst

```diff
@@ -52,17 +57,27 @@ Obtaining and Installing WESTPA
 
 WESTPA is developed and tested on Unix-like operating systems, including Linux and Mac OS X.
 
-We recommend installing WESTPA through conda. See the quick install instructions on our `wiki`_ for how to do this. To install from source (not recommended), please continue reading.
 
 Before installing WESTPA, we recommend you to first install the Python 3 version provided by the latest free `Anaconda Python distribution`_. Create a new python environment for the install with the following::
 
-    conda create -n westpa-2.0 python=3.7
+    conda create -n westpa-2.0 python=3.9
+    conda activate westpa-2.0
+
+Then, we recommend installing WESTPA through conda or pip. Execute either of the following::
+
+    conda install -c conda-forge westpa=2.0
```

Comment:
This doesn't exist yet, maybe we wait on merging this until that's all sorted?

Reply:
It hasn't. Let's wait on this until 2.0's been officially released!